pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -258,3 +295,18 @@ func fitsRequest(podRequest *preFilterState, nodeInfo *framework.NodeInfo, ignor
 
 	return insufficientResources
 }
+
+func isIgnoredResource(name string, ignoredResources, ignoredGroups sets.String) bool {
+	if ignoredResources.Has(name) {
+		return true
+	}
+
+	for group := range ignoredGroups {
+		// We know that name is the extended resource, so it is divided by "/".
+		// group matches the substring before "/".
+		if regexp.MustCompile("^" + group).MatchString(name) {
```

Comment:
`string.HasPrefix` should be way faster. When I was suggesting regex, I was thinking of validation. But it loos like `ValidateLabelName` should be enough.

Reply:
done